app/src/main/java/com/sopt/androidstudy/presentation/home/screens/MainActivity.kt

```diff
@@ -2,21 +2,28 @@ package com.sopt.androidstudy.presentation.home.screens
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.util.Log
 import androidx.activity.viewModels
 import androidx.databinding.DataBindingUtil
 import com.sopt.androidstudy.R
+import com.sopt.androidstudy.data.model.UserData
 import com.sopt.androidstudy.databinding.ActivityMainBinding
 import com.sopt.androidstudy.presentation.home.viewmodels.MyViewModel
 
 class MainActivity : AppCompatActivity() {
     lateinit var binding: ActivityMainBinding
-    private val viewModel: MyViewModel by viewModels()
+    private val _viewModel: MyViewModel by viewModels()
```

Comment:
```suggestion
    private val viewModel: MyViewModel by viewModels()
```

_언더바 빼도 좋을 것 같아요! 기존에 언더바 없는 `viewModel` 변수가 있다면 _를 추가해도 괜찮은데 현재 `viewModel` 변수가 없으니까요~!